### DISCUSSION TITLE
Trouble with "bunx shadcn-svelte@latest init" - Typescript error

### DISCUSSION THREAD

**Discussion by oljeoraklet** [huntabyte/shadcn-svelte#819] [Help] [answered]
I've done all the previous steps installing Sveltekit and Tailwind CSS.

When I try running `bunx shadcn-svelte@latest init`, I get this error:
`This command assumes a SvelteKit project with Tailwind CSS.
If you don't have these, follow the manual steps at https://shadcn-svelte.com/docs/installation.

✔ Running this command will install dependencies and overwrite your existing tailwind.config.[cjs|js|ts] & app.pcss file. Proceed? … yes
✔ Would you like to use TypeScript (recommended)? … no / yes
✔ Which style would you like to use? › Default
✔ Which color would you like to use as base color? › Slate
✔ Where is your global CSS file? … src/app.pcss
✔ Where is your tailwind.config.[cjs|js|ts] located? … tailwind.config.cjs
✔ Configure the import alias for components: … $lib/components
✔ Configure the import alias for utils: … $lib/utils
✔ Write configuration to components.json. Proceed? … yes
Your tailwind.config.cjs has been renamed to tailwind.config.js.

ERROR: Cannot find package 'typescript' imported from /private/tmp/shadcn-svelte@latest--bunx/node_modules/shadcn-svelte/dist/index.js`

Does anyone have any idea what's going on?

Thanks!

**Comment by budivoogt** [comment]
It could be that you're not running the command from the root folder of your project. Use `pwd` to check if you're in the right place. This happened to me once and navigating to the project's root resolved it.

**Reply by oljeoraklet** [reply]
Thank you for the reply! I tried this, but it didn't solve the problem. I tried using 

`npx`
instead and it works with no problem. Curious why it doesn't work with Bun

**Comment by oljeoraklet** [comment, marked as answer]
I solved it. It was just me being stupid, and I hadn't installed the shadcn-svelte package using Bun.
